tensorflow_addons/rnn/cell_test.py

```diff
@@ -175,5 +175,161 @@ def test_NASCell_config(self):
         self.assertEqual(config, restored_config)
 
 
+@test_utils.run_all_in_graph_and_eager_modes
+class LayerNormBasicLSTMCellTest(tf.test.TestCase):
```

Comment:
Should be LayerNormLSTMCell since we have renamed it?

Reply:
Good catch. Done.